content/docs/studio/get-started.md

```diff
@@ -148,3 +157,5 @@ You should now see that a project has been added in your dashboard.
 [create a team]: /doc/studio/user-guide/teams
 [sign up for the **basic**, **teams** or **enterprise** plan]:
   /doc/studio/user-guide/change-team-plan-and-size
+[live-metrics]: /doc/studio/user-guide/projects-and-experiments/live-metrics
```

Comment:
```suggestion
[track live metrics]: /doc/studio/user-guide/projects-and-experiments/live-metrics
```

Reply:
Pls see response to above comment.

Reply:
@jorgeorpinel can I consider this resolved?